fastlane/appstoreres/jetpack_metadata/source/app_name.txt

```diff
@@ -0,0 +1 @@
+Jetpack: WP Security & Speed
```

Comment:
Unless otherwise specified, I copied the `.txt` content from what was already on GlotPress.